.github/workflows/pytest-windows-3.9.yml

```diff
@@ -55,6 +55,11 @@ jobs:
         run: |
           python -m pip --version
           python -m pip install -r requirements/tests.txt
-          python -m pip install . --no-deps
+          python -m pip install -e . --no-deps
       - name: Run tests
-        run: python -m xonsh run-tests.xsh test -- --timeout=240
+        run: python -m xonsh run-tests.xsh test --report-coverage --no-amalgam -- --timeout=240
```

Comment:
Why `--no-amalgam`? These tests need to run with the amalgamated version to prevent bugs like #4289

Reply:
these measure coverage and need actual code.

Reply:
There are other tests running with amalgam

Reply:
So now only 3.9 tests run without amalgamation?

Reply:
Yes. now only linux+3.9 is used for getting coverage